conf/distro/yogurt-vendor-connagtive-provisioning.conf

```diff
@@ -0,0 +1,17 @@
+require conf/distro/yogurt-vendor.conf
+
+DISTRO = "yogurt-vendor-connagtive-provisioning"
+DISTRO_NAME = "Yogurt Vendor Connagtive Provisioning (Phytec Vendor Distribution + Connagtive IoT Device Suite, used for initial provisioning)"
+
+# The distro feature "connagtive-provisioning" is only used for the initial
+# provisioning of a target system in the production stage.
+DISTRO_FEATURES += "connagtive-provisioning"
```

Comment:
Negate the distro feature, remove this one and add a distro feature in yogurt-vendor-connagtive, because all necessary changes will added.